code/modules/paperwork/pen.dm

```diff
@@ -332,3 +332,15 @@
 			P.contact_poison_poisoner = user.name
 			add_attack_logs(user, P, "Poison pen'ed")
 			to_chat(user, "<span class='warning'>You apply the poison to [P].</span>")
+
+// CHAMELEON PEN
+/obj/item/pen/chamel
```

Comment:
Should probably write chameleon out fully, like the other chameleon items (Eg. /obj/item/pen/chameleon)

Reply:
Will fix